exhibits-solr-conf.gemspec

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exhibits-solr-conf/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'exhibits-solr-conf'
```

Comment:
Yes, according to the [Bundler and RubyGems convention](http://guides.rubygems.org/name-your-gem/) if your ruby class/module is `ExhibitsSolrConf` then the gem and file names should be `exhibits_solr_conf`.